README.md

```diff
@@ -102,8 +102,9 @@ Perform the following commands on the terminal and write them down for review,
   - Checkout new branch
   - Merge branch into another
   - See commit history
-  - Rebase and Squash
   - Write a command to clone [this](https://github.com/canopas/web-developer-roadmap-2023) repo
+  - Create two branches, `develop` and `feature`. Add 5 commits to `feature` and 1 commit to `develop`. Merge `feature` into `develop` using squash. The `develop` branch should have `2 commits`, 1 previous, and 1 combined from `feature`. Both branches should have the same changes. 
```

Comment:
- These needs to be applied on each practical, let's mention it.
Changes:
  - Create two branches with practical name for example for practical 1,  `PR-1-staging` and `PR-1-feature`. 
  - Add inital commit to `PR-1-staging`` and minimum 5 commits in `PR-1-feature`. 
  - At the end of practical, development branch should have 2 commits (Initial + combined) with same changes as of feature branch.

Reply:
Minimum 5 commits will be excessive for small practicals. A minimum of 3 will be fine I guess